src/scanpy/_utils/compute/is_constant.py

```diff
@@ -103,22 +103,21 @@ def _(
         else:
             return (a.data == 0).all()
     if axis == 1:
-        return _is_constant_csr_rows(a.data, a.indices, a.indptr, a.shape)
+        return _is_constant_csr_rows(a.data, a.indptr, a.shape)
     elif axis == 0:
         a = a.T.tocsr()
-        return _is_constant_csr_rows(a.data, a.indices, a.indptr, a.shape)
+        return _is_constant_csr_rows(a.data, a.indptr, a.shape)
 
 
 @njit
 def _is_constant_csr_rows(
     data: NDArray[np.number],
-    indices: NDArray[np.integer],
     indptr: NDArray[np.integer],
     shape: tuple[int, int],
-):
+) -> NDArray[np.bool_]:
     n = len(indptr) - 1
     result = np.ones(n, dtype=np.bool_)
-    for i in range(n):
+    for i in numba.prange(n):
```

Comment:
I parallelized this function by replacing the `numba.njit` import with our wrapper. Seems to work fine!